src/display/canvas.js

```diff
@@ -42,6 +42,12 @@ const MAX_SIZE_TO_COMPILE = 1000;
 
 const FULL_CHUNK_HEIGHT = 16;
 
+// Because of https://bugs.chromium.org/p/chromium/issues/detail?id=1170396
+// some curves aren't rendered correctly.
+// Multiplying lineWidth by this factor should help to have "correct"
+// rendering with no artifacts.
```

Comment:
Should we mention here that this constant can be removed entirely once the upstream bug is fixed? Or should this patch basically be reverted in that case, given that the 1.000001 number was already in use before this patch if `absDet > Number.EPSILON`?

Reply:
In the updated patch the multiplication by the constant is chrome-only.